src/app/bills/bill-view.ts

```diff
@@ -0,0 +1,52 @@
+import { Bill } from './bill';
+
+export class BillView {
+  constructor(private bill: Bill) { }
+
+  get id() { return this.bill.id; }
+  get cashback() { return this.bill.cashback; }
+  get uid() { return this.bill.uid; }
+  get vat() { return this.bill.vat; }
+  get workHours() { return this.bill.workHours; }
+
+  get deleted() { return this.bill.deleted; }
+  get finished() { return this.bill.finished; }
+  get paid() { return this.bill.paid; }
```

Comment:
Clean Code: the order should make sense when reading it. Thus, the order should be:
get paid()
get finished()
get deleted()